CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 
 All notable changes to the kdb extension will be documented in this file.
 
+# v1.5.0
+
+This release requires at least version 1.86.0 of VS Code
+
+### Enhancements
+
+- New `Workbook`files concept with extension `*.kdb.q` (for q) or `*.kdb.py` (for Python) with selectable connection
+- Data sources are created in current workspace folder as `*.kdb.json` files and associated connection can be selected
+- New DATASOURCES and WORKBOOKS views in KX tab which lists workbook and data source files per workspace
+- Multiple logged in connections support, active connection can be selected among logged in connections
```

Comment:
Can we replace the first 4 bullet points with this please

- Ability to have more than one connection open simultaneously. q and Python files run against the active connection which can be chosen from the open connections.
- There is a new concept of `Workbook` files with extension `*.kdb.q` (for q) or `*.kdb.py` (for Python) created in the current workspace folder, that can be associated with a specific connection rather than just running against the active connection.
- Data sources are created in current workspace folder as `*.kdb.json` files and can be associated with a specific connection rather than just running against the active connection.
- New DATASOURCES and WORKBOOKS views in KX tab which lists workbook and data source files per workspace.